src/content/workers-ai-models/whisper-large-v3-turbo.json

```
@@ -44,7 +44,7 @@
                 },
                 "prefix": {
                     "type": "string",
-                    "description": "The prefix it appended the the beginning of the output of the transcription and can guide the transcription result."
+                    "description": "The prefix it appended the beginning of the output of the transcription and can guide the transcription result."
```

```suggestion
                    "description": "The prefix appended at the beginning of the output of the transcription to guide the transcription result."
```
@craigsdennis Since this is extracted info, I don't know how to fix this small description issue at the source.
Also, can you check if my proposed sentence makes (more) sense?

---

Note to readers: This is also being fixed internally.